evo_researcher/benchmark/benchmark.py

```diff
@@ -223,7 +261,7 @@ def get_markets_summary(self) -> t.Dict[str, t.List[str]]:
 
         for agent in [a.agent_name for a in self.registered_agents]:
             markets_summary[f"{agent} p_yes"] = [
-                self.get_prediction(agent_name=agent, question=q).p_yes
+                p.completion_prediction.p_yes if (p := self.get_prediction(agent_name=agent, question=q)).completion_prediction else None
```

Comment:
I think it would be great to differentiate here too between a None from an exception vs. a None from 'evaluated as unanswerable'

Reply:
Added!